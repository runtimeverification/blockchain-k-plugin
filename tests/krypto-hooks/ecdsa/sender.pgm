#sender(
    b"\xc8\xc7\x95s/\xa0F\xc4H\xd6\xa9\x1c\x19\x11\xcaZ\x87\xcb-\x17\x13\xcb\x87\x08\xb9gF\x01\xda\xe5\x13\r",
    27,
    b"\xba\xf8\xa8\xf5\x93%\x06L\xb7\x94A\xe4\xe7\x83P\xce\xdc\xb7\xf7c\x17\xc4\xf8\xd9\xed2\x10%\xdbR\x91k",
    b"L\xc9\x88\xdb\x9c\xecW\xed\xa4V\xbb$\"\x01\xbbA\xc7#\xf2B\x7f\xf7P\x12?\x11\xafku\x95G\xe2"
)
