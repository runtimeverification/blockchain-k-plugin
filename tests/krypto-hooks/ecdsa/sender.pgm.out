<k>
  966588469268559010541288244128342317224451555083 ~> .
</k>
